source/clear-linux/guides/network/network-bonding.rst

```diff
@@ -1,31 +1,41 @@
 .. _network-bonding:
 
-Combine multiple interfaces
-###########################
+Combine multiple interfaces with network bonding
+################################################
 
-Network bonding is a technique for combining multiple network interfaces into
-a single, logical interface, providing some redundancy and bandwidth
-aggregation.
+Network bonding combines multiple network interfaces into a single logical
+interface to provide redundancy and bandwidth aggregation.
 
-|CLOSIA| includes the bonding_ and team_ drivers. The guide example provided
-below shows how to configure systemd to use the ``bonding`` driver.
+|CLOSIA| includes Linux bonding_ and team_ drivers. This guide describes how
+to configure systemd to use the `bonding` driver.
+
+The example demonstrates how to:
+
+*  Bond all four ports of a quad-port NIC in `802.3ad` mode.
+
+*  Enable jumbo frames to optimize large data transfers on the local network.
+
+You may need to configure your NICs and network switch to support `802.3ad`
```

Comment:
The NICs and network switch must have specific hardware _support_ for each of these features.

Our example includes _configuring_ the NICs for both of these features.

Configuring the switch is left as an exercise for the reader.

Reply:
Thanks for the feedback. Please let me know if the updated text is OK.